python/spear/enhanced_input_service.py

```diff
@@ -16,6 +16,10 @@ def inject_input_for_actor(self, actor, input_action_name, trigger_event, input_
         self._entry_point_caller.call("enhanced_input_service.inject_input_for_actor",
             actor, input_action_name, trigger_event, spear.to_json_string(input_action_value), spear.to_json_string(input_action_instance), modifiers, triggers, verbose)
 
+    def inject_input_for_blueprint_actor(self, actor, input_action_name, trigger_event, input_action_value, input_action_instance, modifiers=[], triggers=[], verbose=False):
```

Comment:
Remove until we agree on the design.

Reply:
removed.

Reply:
This function, that we agreed to remove, is still present in the PR after you said three weeks ago that you removed it. Please don't make me triple-check your code like this.

Reply:
fixed